test/general/interface_check.pass.cpp

```diff
@@ -22,7 +22,6 @@
 #include <functional>
 #include <iterator>
 #include <vector>
-#include "../support/pstl_test_config.h"
```

Comment:
This is not correct to remove config file from tests. External users don't include `support` folder to CPATH by default. If they run tests separately, tests will be skipped because macros from config file are not defined. Test system on our side also will skip these tests and coverage is affected. Could you please provide more details why it's added to CPATH on infrastructure side? As for me it seems incorrect and can be fixed on infrastructure side.

Reply:
I don't quite understand the PR description: how setting `CPATH` can replace `#include` statement?

`#include` statement lets preprocessor know that this header _must_ be included during the compilation, while setting `CPATH` just gives compiler know additional search paths for included headers, so `CPATH` is rather alternative to compiler keys like `-I`, `-isystem`, etc.

Reply:
@AlexVeprev Because for our regular testing, it will only copy source code file and support folder to local, and use command such as:
dpcpp  -DPSTL_USE_PARALLEL_POLICIES=0 interface_check.pass.cpp -o interface_check.pass.exe -ltbb -lOpenCL. 
So the relative path in test code line will make it compile fail.